.travis.yml

```diff
@@ -1,17 +1,19 @@
-dist: xenial
+dist: bionic
+language: python
+
 addons:
-  postgresql: "9.6"
+  postgresql: "9.4"
   apt:
     packages:
-     - postgresql-9.6-postgis-2.3
-language: python
-sudo: false
-cache: pip
+      - postgresql-9.4-postgis-2.4
```

Comment:
why downgrade to a lower version of postgres? [9.4 is supported until february 13, 2020](https://www.postgresql.org/support/versioning/)

Reply:
Agreed. And using Travis CI docs found a simpler solution.